generation/remap-handles.rsp

```diff
@@ -11,6 +11,8 @@ LDT_ENTRY*=@void*
 PCONTEXT=@void*
 PIMAGE_NT_HEADERS=@void*
 PMemoryAllocator*=@void*
+PMINIDUMP_EXCEPTION_INFORMATION=@void*
+PMINIDUMP_USER_STREAM_INFORMATION=@void*
```

Comment:
Not sure I understand these remappings?

We have the concrete type `MINIDUMP_EXCEPTION_INFORMATION*` and `MINIDUMP_USER_STREAM_INFORMATION*`

Similarly I'd expect we could remove the handle remapping for `PMINIDUMP_CALLBACK_INFORMATION` and now make it `MINIDUMP_CALLBACK_INFORMATION*`